repose-aggregator/components/filters/keystone-v2/src/main/resources/META-INF/schema/config/keystone-v2.xsd

```diff
@@ -108,6 +108,13 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="set-rackers-groups" type="xs:string" use="optional">
```

Comment:
Yes, well I would want to pull the set-groups attribute into that new element. Which means either moving it, which isn't backwards compatible, or having multiple ways to configure it, which is ugly and a pain.